sql/hive/src/test/scala/org/apache/spark/sql/hive/StatisticsSuite.scala

```diff
@@ -178,6 +179,7 @@ class StatisticsSuite extends QueryTest with BeforeAndAfterAll {
         sql(s"""SET ${SQLConf.AUTO_BROADCASTJOIN_THRESHOLD}=$tmp""")
       }
 
+      conf.setConf("spark.sql.autoSortMergeJoin", "true")
```

Comment:
Since the default value of it is false, seems it is not safe to turn it on at here? Actually, as I commented above, we do not need to set this conf property, right?